libecole/src/dynamics/branching-gub.cpp

```diff
@@ -0,0 +1,198 @@
+/***********************************************************
+ *  Implementation of SCIPbranchGUB (aim to merge in SCIP) *
+ ***********************************************************/
+
+#include <cassert>
+
+#include <fmt/format.h>
+
+#include <scip/cons_linear.h>
+#include <scip/debug.h>
+#include <scip/scip.h>
+#include <scip/scip_branch.h>
+#include <scip/scip_numerics.h>
+#include <scip/struct_scip.h>
+#include <scip/struct_var.h>
+
+namespace {
+
+SCIP_RETCODE SCIPbranchGUB_add_child(
+	SCIP* scip,
+	SCIP_Real priority,
+	SCIP_Real estimate,
+	SCIP_VAR** vars,
+	SCIP_Real* ones,
```

Comment:
Why is this argument called ones ? Does it have to be a vector of ones ? If so, why is it an argument ?